### ISSUE TITLE
[Flaky CI] Cluster Network Addons Operator when installed from master release and when NodeNetworkConfig with supported spec is created stays in Available condition while the operator is being removed and redeployed

### ISSUE THREAD

**Issue by omeryahud** [kubevirt/cluster-network-addons-operator#461]
/triage build-officer
/kind bug

**What happened**:
Flakefinder reported the flaky test
```
Cluster Network Addons Operator when installed from master release and when NodeNetworkConfig with supported spec is created stays in Available condition while the operator is being removed and redeployed
```
In these Prow jobs:
-  [Timed out](https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_cluster-network-addons-operator/459/pull-e2e-cluster-network-addons-operator-lifecycle-k8s/1281497105831038976)
- [RBAC issue](https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_cluster-network-addons-operator/448/pull-e2e-cluster-network-addons-operator-lifecycle-k8s/1280849464193454080)

**What you expected to happen**:
The test should pass

**How to reproduce it (as minimally and precisely as possible)**:
See Prow jobs

**Anything else we need to know?**:

**Environment**:

**Comment by RamLavi** [comment]
`s: "Secret \"kubemacpool-service\" has been found",` error should be fixed in the latest KMP (0.16.0) & KNMSTATE(0.23.0) bumps - due to a fix in kaw [fix](https://github.com/qinqon/kube-admission-webhook/pull/32)

**Comment by RamLavi** [comment]
@phoracek I think this issue is already resolved. If so, can you close it?
